certexchange/client.go

```diff
@@ -136,3 +139,34 @@ func (c *Client) Request(ctx context.Context, p peer.ID, req *Request) (_rh *Res
 	}()
 	return &resp, ch, nil
 }
+
+func (c *Client) FindInitialPowerTable(ctx context.Context, powerTableCID cid.Cid) (gpbft.PowerEntries, error) {
+	request := Request{
+		FirstInstance:     0,
+		Limit:             0,
+		IncludePowerTable: true,
+	}
+
+	for {
+		for _, p := range c.Host.Network().Peers() {
```

Comment:
Check if the peer supports the protocol before asking. Otherwise, we'll block on multistream each time.